resources/views/partials/content-page-stories.blade.php

```diff
@@ -0,0 +1,30 @@
+<article class="card pcc-story-card">
+    <figure>
+        @if(has_post_thumbnail())
+          {!! get_the_post_thumbnail(get_the_ID(), 'small') !!}
+        @else
+        <div class="placeholder-wrap">
+            @svg('mark', ['class' => 'placeholder'])
+        </div>
+        @endif
+    </figure>
+    <div class="pcc-story__details">
+        <header class="text">
+            <h2 class="title">
+                <a href="{{ get_permalink() }}">{!! get_the_title() !!}</a>
+            </h2>
+        </header>
+        <div class="pcc-story__meta">
+            <div class="card__region">
+                <span class="screen-reader-text">{{ __('Region:','pcc') }} </span>
+                <svg class="icon icon--location" aria-hidden="true" viewBox="0 0 20 20" focusable="false">
+                    <use href="{{ App\asset_path('images/location.svg#location') }}" />
+                </svg>
```

Comment:
Instead of doing this, use the `@svg` directive (see line 7 above for an example). This isn't working for some reason but I know that will.